pkg/datastore/mysql/ensurer/indexes.sql

```diff
@@ -78,6 +78,10 @@ CREATE INDEX deployment_status_updated_at_desc ON Deployment (Status, UpdatedAt
 ALTER TABLE Deployment ADD COLUMN PipedId VARCHAR(36) GENERATED ALWAYS AS (data->>"$.piped_id") VIRTUAL NOT NULL;
 CREATE INDEX deployment_piped_id ON Deployment (PipedId);
 
+-- index on `CompletedAt` ASC
+ALTER TABLE Deployment ADD COLUMN CompletedAt INT(11) GENERATED ALWAYS AS (data->>"$.completed_at") VIRTUAL NULL;
+CREATE INDEX deployment_completed_at ON Deployment (CompletedAt);
```

Comment:
nits, I don't think we need this index, instead of making this, maybe multiple columns index on CompletedAt and Id column is better, since this CompletedAt column is not be referred independently but in query with other columns. Should use `explain` query and check it out on MySQL cli.

Reply:
Thank you, I'll check it.